### DISCUSSION TITLE
Unit 10: The Finale!

### DISCUSSION THREAD

**Discussion by rasbt** [Lightning-AI/dl-fundamentals#59] [Announcements]
I am excited to share that Unit 10 is now live!

https://lightning.ai/pages/courses/deep-learning-fundamentals/10.0-overview-the-finale-our-next-steps-after-ai-model-training/

If you have been following along, I hope you enjoyed the journey from backpropagation to implementing computer vision models and LLMs in PyTorch!

My goal with this course was to provide a modern take on learning deep learning & AI using open-source libraries. Moreover, I tried to be concise but also go beyond the typical topics covered in university classes.

What was also particularly important to me was to also include important topics that are super relevant in practice: learning rate schedulers, mixed-precision training, distributed multi-GPU training strategies, and more!

**Comment by rsousa95** [comment]
I've just finished the course! I'll have to say, I really liked it. I found it quite good and I learned a lot. Thank you for taking the time to do this, Sebastian! Just a question, is it possible to get some kind of certificate of completion? I'd love to put this on my resume or linkedin. Also, are there any plans to do another course in the future? I'd definitely be up to that.

**Reply by rasbt** [reply]
@rsousa95 Whoa, exciting to hear that you completed the whole course, and it's nice to hear that you liked it! 

By the way, a completion badge is coming, we have been actively working on it, and I hope it will be ready to roll out some time next week.
